lib/Data/ValueProxy.js

```diff
@@ -231,14 +230,15 @@ export class ValueProxy extends Events {
      */
     static fromModel(obj, key) {
         return new ValueProxy({
+            generation: obj._generation,
```

Comment:
`getGeneration` method will be exposed to every subclass of `Events`, which is not good idea for `Model`.